components/Currency/CurrencyIcon/CurrencyIcon.tsx

```diff
@@ -93,12 +69,36 @@ export const CurrencyIcon: FC<CurrencyIconProps> = ({ currencyKey, type, ...rest
 								? synthetixTokenListMap[currencyKey].logoURI
 								: getSynthIcon(currencyKey as CurrencyKey)
 						}
-						onError={() => setThirdFallbackError(true)}
+						onError={() => setFirstFallbackError(true)}
 						{...props}
 						alt={currencyKey}
 					/>
 				);
 		}
+	} else if (
+		OneInchTokenListMap != null &&
+		OneInchTokenListMap[currencyKey] != null &&
```

Comment:
(OneInchTokenListMap?[currencyKey] ?? null) != null

Reply:
np: can merge without changing

Reply:
TS still thinks it could be null in that case 

![image](https://user-images.githubusercontent.com/4015977/127935778-0b3580b3-2fb3-4758-a88a-82a27e40dfc2.png)